components/BottomNavigation/src/ColorThemer/MDCBottomNavigationBarColorThemer.m

```diff
@@ -16,8 +16,19 @@
 
 #import "MDCBottomNavigationBarColorThemer.h"
 
+static const CGFloat kMDCBottomNavigationColorThemerUnselectedOpacity = 0.6f;
```

Comment:
Can drop the `MDCBottomNavigationColorThemer` part.

Reply:
Done, thanks!